### PR TITLE
TFAC-222 Creating InviteFriends Component

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: src/components/InviteFriends.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    textFieldRef.current.select()
+  }
+
+  const getHelperText = () => {
+    if (helperText !== '') {
+      return helperText
+    }
+
+    return get(user, 'username')
+      ? `and you'll get 350 Hearts for every person who joins!`
```
Comment (amaliwanag2) [inline src/components/InviteFriends.js:30]: this copy is out of date.  Not sure what it should be tho

Comment (jedtan) [reply src/components/InviteFriends.js:30]: Sounds reasonable. @kmjennison thoughts?

Comment (kmjennison) [reply src/components/InviteFriends.js:30]: @jedtan We actually don't need any text/label at the moment, so remove the default text altogether.

**Review by kmjennison — CHANGES_REQUESTED** [review]
Looks good except for the unneeded text and label.

**Pull request merged**